1. Implemented UI chnages

Abdulrehman53 opened the pull request:
Implemented UI changes

Commit:
1. Implemented UI chnages

Commit:
Fixed page broken issue

sakinaboriwala commented on lib/screens/categorylisting.dart:
```diff
@@ -29,6 +29,7 @@ class _CategoryListingState extends State<CategoryListing> {
   Size _deviceSize;
   bool _isLoading = true;
   int level = 0;
+  bool isLevelClicked = false;
```
Why have you added this variable? If it's not in use, please remove it.

sakinaboriwala commented on lib/screens/home.dart:
```diff
@@ -64,11 +64,12 @@ class _HomeScreenState extends State<HomeScreen> {
       return Scaffold(
         appBar: AppBar(
-          centerTitle: true,
+          centerTitle: false,
```
For HomePage, title should be centered for similar behaviour with Android, iOS

Commit:
UI Changes

Commit:
1. Resolved search issue

Commit:
1. Enable search when search button is pressed
2. Reduce size of today's deal item
3. Enable search when search button is pressed

sakinaboriwala commented on lib/screens/search.dart:
```diff
@@ -71,15 +71,7 @@ class _ProductSearchState extends State<ProductSearch> {
-                      searchProduct();
-                    });
-                  },
+                  onChanged: (value) {},
```
The value that user enters should be saved in the "slug" variable and then sent with the API call.

sakinaboriwala replied:
Please set this currently same search results are shown always as you are sending just a space in the API.

sakinaboriwala reviewed (changes requested):
Please resolve the search Issue.

Commit:
Resolved search issue

Commit:
1. Resolved progress bar issue when go to detail page from search product

sakinaboriwala reviewed (approved):
LGTM
